sys/include/net/gnrc/tcp.h

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (C) 2015 Simon Brummer
+ * Copyright (C) 2017 Simon Brummer
```

Comment:
Well I wrote it during my bachelor thesis without getting paid by my university so I would argue the copyright goes to me. ;)

I'll update the the copyright year accordingly.